### PR TITLE
Example Fix - Update app id button

### PR DESCRIPTION
**Pull request by jkasten2**
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-ios-sdk/971)

### CODE REVIEW

File: iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m [hunk 1]
```diff
@@ -98,20 +98,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
-    if (![newAppId isEqualToString:onesignalAppId]) {
-        [self setOneSignalAppId:newAppId];
-        onesignalAppId = newAppId;
+    NSString* userDefinedAppId = [[NSUserDefaults standardUserDefaults] objectForKey:ONESIGNAL_APP_ID_KEY_FOR_TESTING];
+    if (userDefinedAppId) {
+        return userDefinedAppId;
     }
-
-    return onesignalAppId;
+    return @"1688d8f2-da7f-4815-8ee3-9d13788482c8";
```
Comment (Jeasmine) [inline iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m:105]: Is ok to have this id by default?

**Review by Jeasmine — APPROVED** [review]
LGTM nit comment

File: iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m (hunk 1 above)
Comment (emawby) [reply iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m:105]: LGTM other than this ID

**Review by jkasten2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-ios-sdk/971)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @emawby and @Jeasmine)

___
*[iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m, line 105 at r3](https://reviewable.io/reviews/onesignal/onesignal-ios-sdk/971#-MgNlnBxdK_bTc5dRxuY-r3-105:-MgSCJ7S-SnV8REBIXcm:bfe5v8m) ([raw file](https://github.com/onesignal/onesignal-ios-sdk/blob/4cfd0598c2f21a9e368fe3fa4471a379c5e7304d/iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m#L105)):*
<details><summary><i>Previously, emawby (Elliot Mawby) wrote…</i></summary><blockquote>

LGTM other than this ID
</blockquote></details>

Good catch, I'll put this defaulting back to `"0ba9731b-33bd-43f4-8b59-61172e27447d"` as it was this before.

**Review by jkasten2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-ios-sdk/971)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @emawby and @Jeasmine)

___
*[iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m, line 105 at r3](https://reviewable.io/reviews/onesignal/onesignal-ios-sdk/971#-MgNlnBxdK_bTc5dRxuY-r3-105:-MgSNeKNE6I6XHQRblQF:bps9s4t) ([raw file](https://github.com/onesignal/onesignal-ios-sdk/blob/4cfd0598c2f21a9e368fe3fa4471a379c5e7304d/iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m#L105)):*
<details><summary><i>Previously, jkasten2 (Josh Kasten) wrote…</i></summary><blockquote>

Good catch, I'll put this defaulting back to `"0ba9731b-33bd-43f4-8b59-61172e27447d"` as it was this before.
</blockquote></details>

Fixed!
Also moved the id into a `ONESIGNAL_APP_ID_DEFAULT` definition so it is more clear it is a default value.

**Pull request merged**